### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by naveen521kk**
Update Readme with the example.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -180,23 +180,23 @@ depending on whether you want to use HTTPS or SSH. Finally, run the
...
 
 To run your first example Scene, you can run the following commands:
 For users:
-1. Download the `example_scenes.py` file from [GitHub](https://github.com/ManimCommunity/manim/blob/master/example_scenes.py)
+
+1. Download the `example_scenes.py` file from [GitHub](https://github.com/ManimCommunity/manim/blob/master/example_scenes/basic.py)
 2. Run the following command:
 ```sh
-manim example_scenes.py SquareToCircle -pl
+manim basic.py SquareToCircle -pl
```
Comment (kilacoda) [inline README.md:194]: It's `example_scenes/basic.py`

Comment (naveen521kk) [reply README.md:194]: When downloaded the user will store like that.

File: README.md [hunk 2]
```diff
@@ -180,23 +180,23 @@ depending on whether you want to use HTTPS or SSH. Finally, run the
 following:
 
 ```sh
```
Comment (naveen521kk) [inline README.md:182]: How about changing `sh` to `bash`?

Comment (PgBiel) [reply README.md:182]: will this effectively change anything at all?

**Comment by naveen521kk** [comment]
How about adding documentation about using `venv`? This would help users seperate their *global* python installation and *manim*.

**Comment by leotrs** [comment]
@naveen521kk I'm in favor of adding a few lines to that effect in the README. However, I think this could be in a different issue or PR than this one.

File: README.md (hunk 1 above)
Comment (PgBiel) [reply README.md:194]: huh? @naveen521kk

**Comment by leotrs** [comment]
@naveen521kk thanks for the latest changes. Just gave my review, but we need one more. Perhaps @kilacoda?

File: README.md [hunk 3]
```diff
@@ -180,23 +180,23 @@ depending on whether you want to use HTTPS or SSH. Finally, run the
...
 2. Run the following command:
 ```sh
-manim example_scenes.py SquareToCircle -pl
+manim example_scenes/basic.py SquareToCircle -pl
 ```
 1. Run the following command:
 For developers:
 ```sh
-python3 manim.py example_scenes.py SquareToCircle -pl
+python3 manim.py example_scenes\basic.py SquareToCircle -pl
```
Comment (kilacoda) [inline README.md:199]: This is wrong though? We don't have `manim.py` now.

Comment (leotrs) [reply README.md:199]: @kilacoda could you suggest the correct alternative? Or perhaps @XorUnison?

Comment (kilacoda) [reply README.md:199]: It's the same for users and devs now.

Comment (Aathish04) [reply README.md:199]: It should be `manim example_scenes.py SquareToCircle -pl`

That's after installing with `pip install -e .`

Comment (naveen521kk) [reply README.md:199]: Can we remove those lines then?

Comment (Aathish04) [reply README.md:199]: Yes, you should edit them.

File: README.md [hunk 4]
```diff
@@ -180,17 +180,29 @@ depending on whether you want to use HTTPS or SSH. Finally, run the
...
+
+### For developers:
+
+- Run the following command to install manim:
+```sh
+pip install -e .
+```
+- Then run the following command below:
+```sh
 manim example_scenes.py SquareToCircle -pl
```
Comment (kilacoda) [inline README.md:206]: It's `manim example_scenes/basic.py SquareToCircle -pl`

Comment (PgBiel) [reply README.md:206]: @kilacoda add the suggestion thing

Comment (kilacoda) [reply README.md:206]: Uhh... how?

Comment (kilacoda) [reply README.md:206]: Oh ok nvm found it

Comment (kilacoda) [reply README.md]: Done

**Pull request merged**